content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -436,192 +623,275 @@ Resources are deleted in two phases: 1) finalization, and 2) removal.
+resources, and **deletecollection** allows deleting multiple resources.
 
 ## Dry-run
 
  {{< feature-state for_k8s_version="v1.18" state="stable" >}}
 
-The modifying verbs (`POST`, `PUT`, `PATCH`, and `DELETE`) can accept requests in a _dry run_ mode. Dry run mode helps to evaluate a request through the typical request stages (admission chain, validation, merge conflicts) up until persisting objects to storage. The response body for the request is as close as possible to a non-dry-run response. The system guarantees that dry-run requests will not be persisted in storage or have any other side effects.
+When you use an HTTP verbs that can modify resources (`POST`, `PUT`, `PATCH`, and
+`DELETE`), you can submit your request in a _dry run_ mode. Dry run mode helps to
+evaluate a request through the typical request stages (admission chain, validation,
+merge conflicts) up until persisting objects to storage. The response body for the
+request is as close as possible to a non-dry-run response. Kubernetes guarantees
+that dry-run requests will not be persisted in storage or have any other side effects.
 
 
 ### Make a dry-run request
 
-Dry-run is triggered by setting the `dryRun` query parameter. This parameter is a string, working as an enum, and the only accepted values are:
+Dry-run is triggered by setting the `dryRun` query parameter. This parameter is a
+string, working as an enum, and the only accepted values are:
 
-* `All`: Every stage runs as normal, except for the final storage stage. Admission controllers are run to check that the request is valid, mutating controllers mutate the request, merge is performed on `PATCH`, fields are defaulted, and schema validation occurs. The changes are not persisted to the underlying storage, but the final object which would have been persisted is still returned to the user, along with the normal status code. If the request would trigger an admission controller which would have side effects, the request will be failed rather than risk an unwanted side effect. All built in admission control plugins support dry-run. Additionally, admission webhooks can declare in their [configuration object](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#webhook-v1beta1-admissionregistration-k8s-io) that they do not have side effects by setting the sideEffects field to "None". If a webhook actually does have side effects, then the sideEffects field should be set to "NoneOnDryRun", and the webhook should also be modified to understand the `DryRun` field in AdmissionReview, and prevent side effects on dry-run requests.
-* Leave the value empty, which is also the default: Keep the default modifying behavior.
+[empty]
+: Allow side effects. You request this with a query string such as `?dryRun`
+  or `?dryRun&pretty=true`. The response is the final object that would have been
+  persisted, or an error if the request could not be fulfilled.
 
-For example:
+`All`
```

Comment:
Check the preview. The text in the description list is rendering differently. The first element uses one type of font compared to the second element (formatted string "All").

Reply:
`All` is a literal value. I changed [empty] to [no value set]; is that more clear?